src/main/java/uk/gov/companieshouse/service/payment/RefundService.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.companieshouse.service.payment;
+
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.client.PaymentsClient;
+import uk.gov.companieshouse.model.dto.payment.RefundRequest;
+import uk.gov.companieshouse.model.dto.payment.RefundResponse;
+
+import java.util.Optional;
+
+@Service
+public class RefundService {
```

Comment:
Yeah include the mapping in here too - otherwise it's an unnecessary abstraction